lib/src/controls/navigation/tab_view.dart

```diff
@@ -928,9 +940,10 @@ class __TabState extends State<_Tab>
                   ),
             decoration: BoxDecoration(
               borderRadius: borderRadius,
-
               // if selected, the background is painted by _TabPainter
-              color: widget.selected ? null : backgroundColor,
+              color: widget.selected
+                  ? widget.tab.selectedColor
+                  : widget.tab.color ?? backgroundColor,
```

Comment:
The resolved `backgroundColor` must be applied even if `tab.selectedColor` is null.

```suggestion
              color: (widget.selected
                  ? widget.tab.selectedColor
                  : widget.tab.color) ?? backgroundColor,
```